File: exercise_20210228/exercise.py
# 計算 1 + 2 + ... + 50 的總和
sum = 0
for x in range(1, 51):
    sum += x

print(sum)

# 印出 99 乘法表
for i in range(1, 10):
    for j in range(1, 10):
        product = i * j
        print(i, " x ", j, " = ", product)

File: exercise_20210228/exercise.js
// 計算 1 + 2 + ... + 50 的總和
let sum = 0;
for (let n = 1; n <= 50; n++) {
    sum += n;
}
console.log(sum);

// 印出 99 乘法表
for (let x = 1; x <= 9; x++) {
    for (let y = 1; y <= 9; y++) {
        product = x * y;
        console.log(`${x} x ${y} = ${product}`);
    }
}